[2024-11-01T00:30:00.268+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-01T00:30:00.281+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.create_table manual__2024-11-01T00:29:56.389852+00:00 [queued]>
[2024-11-01T00:30:00.288+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.create_table manual__2024-11-01T00:29:56.389852+00:00 [queued]>
[2024-11-01T00:30:00.289+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-01T00:30:00.300+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): create_table> on 2024-11-01 00:29:56.389852+00:00
[2024-11-01T00:30:00.306+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=252) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-01T00:30:00.307+0000] {standard_task_runner.py:72} INFO - Started process 253 to run task
[2024-11-01T00:30:00.307+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'create_table', 'manual__2024-11-01T00:29:56.389852+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/Etl_weather.py', '--cfg-path', '/tmp/tmp_yznwvcq']
[2024-11-01T00:30:00.308+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask create_table
[2024-11-01T00:30:00.344+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.create_table manual__2024-11-01T00:29:56.389852+00:00 [running]> on host 915ee6de4e26
[2024-11-01T00:30:00.407+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T00:29:56.389852+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-01T00:29:56.389852+00:00'
[2024-11-01T00:30:00.408+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-01T00:30:00.418+0000] {sql.py:266} INFO - Executing: 
         CREATE TABLE IF NOT EXISTS weather_medellin (
             city varchar(255),
             temp float,
             temp_min float,
             temp_max float,
             pressure integer,
             humidity integer,
             time timestamp
         );
         
[2024-11-01T00:30:00.426+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-01T00:30:00.433+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-01T00:30:00.465+0000] {sql.py:509} INFO - Running statement: 
         CREATE TABLE IF NOT EXISTS weather_medellin (
             city varchar(255),
             temp float,
             temp_min float,
             temp_max float,
             pressure integer,
             humidity integer,
             time timestamp
         );
         , parameters: None
[2024-11-01T00:30:00.475+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-01T00:30:00.475+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=create_table, run_id=manual__2024-11-01T00:29:56.389852+00:00, execution_date=20241101T002956, start_date=20241101T003000, end_date=20241101T003000
[2024-11-01T00:30:00.523+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-01T00:30:00.537+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-01T00:30:00.539+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
